cloudbuild.yaml

```diff
@@ -0,0 +1,28 @@
+steps:
+- name: 'gcr.io/cloud-builders/docker'
+  args: [
+    'build',
+    '-t', 'us-central1-docker.pkg.dev/$PROJECT_ID/usage_accountant/image:$COMMIT_SHA',
+    '-t', 'us-central1-docker.pkg.dev/$PROJECT_ID/usage_accountant/image:latest',
+    '--build-arg',
+    'VERSION=$COMMIT_SHA',
```

Comment:
You do not have this argument defined in your Dokerfile. Is this intentional ?

Reply:
It's suppose to be substituted at build time, https://cloud.google.com/build/docs/configuring-builds/substitute-variable-values